diff --git a/openpose.py b/openpose.py
index dd128e1..9ac780c 100644
--- a/openpose.py
+++ b/openpose.py
@@ -27,6 +27,8 @@ inWidth = args.width
 inHeight = args.height
 
 net = cv.dnn.readNetFromTensorflow("graph_opt.pb")
+net.setPreferableBackend(cv.dnn.DNN_BACKEND_CUDA)
+net.setPreferableTarget(cv.dnn.DNN_TARGET_CUDA)
 
 cap = cv.VideoCapture(args.input if args.input else 0)
 
@@ -77,4 +79,4 @@ while cv.waitKey(1) < 0:
     freq = cv.getTickFrequency() / 1000
     cv.putText(frame, '%.2fms' % (t / freq), (10, 20), cv.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 0))
 
-    cv.imshow('OpenPose using OpenCV', frame)
\ No newline at end of file
+    cv.imshow('OpenPose using OpenCV', frame)
